x-pack/test/functional/apps/endpoint/management.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const pageObjects = getPageObjects(['common', 'endpoint']);
+  const esArchiver = getService('esArchiver');
+  const testSubjects = getService('testSubjects');
+
+  describe('Endpoint Management List', function() {
+    this.tags('ciGroup7');
+    before(async () => {
+      await esArchiver.load('endpoint/endpoints/api_feature');
+      await pageObjects.common.navigateToUrlWithBrowserHistory('endpoint', '/management');
+    });
+
+    it('finds title', async () => {
+      const title = await testSubjects.getVisibleText('managementViewTitle');
+      expect(title).to.equal('Hosts');
+    });
+
+    it('displays table data', async () => {
+      const data = await pageObjects.endpoint.getManagementTableData();
+      [
+        'Hostnamecadmann-4.example.com',
+        'PolicyPolicy Name',
+        'Policy StatusPolicy Status',
+        'Alerts0',
+        'Operating Systemwindows 10.0',
+        'IP Address10.192.213.130, 10.70.28.129',
+        'Sensor Versionversion',
+        'Last Activexxxx',
+      ].forEach((cellValue, index) => {
+        expect(data[1][index]).to.equal(cellValue);
+      });
+    });
+
+    after(async () => {
+      await esArchiver.unload('endpoint/endpoints/api_feature');
+    });
+  });
+};
```

Comment:
@charlie-pichette @parkiino - I'm adding a note here on the tests we decided to execute:

**Automated**
- [x] Endpoint list component renders correctly on the page when Endpoint app is enabled
- [x] columns are rendered in proper order

**Manual**
- [ ] Verify max size in all fields (will be tested manually)
- [ ] Verify max size when all fields have their max size (will be tested manually)

Reply:
@parkiino @paul-tavares - I am verifying the max size in each field on Endpoint List. I'm seeing the hostname wrap and the other column field data show up lower down on the EUI table. What is our expected behavior for when the column contains a very large value?
<img width="222" alt="Screen Shot 2020-02-12 at 9 48 38 PM" src="https://user-images.githubusercontent.com/56604066/74405556-3a0b7b00-4de2-11ea-9d25-7a34614d5300.png">
<img width="1445" alt="Screen Shot 2020-02-12 at 9 49 11 PM" src="https://user-images.githubusercontent.com/56604066/74405558-3bd53e80-4de2-11ea-8e4a-6dce1729037c.png">
<img width="499" alt="Screen Shot 2020-02-12 at 9 49 34 PM" src="https://user-images.githubusercontent.com/56604066/74405560-3c6dd500-4de2-11ea-9564-9606a3e0e026.png">

Reply:
@parkiino @paul-tavares - also working on verifying when more than one field contains a very large value. Tested 3 fields and got this:
<img width="1325" alt="Screen Shot 2020-02-12 at 10 18 38 PM" src="https://user-images.githubusercontent.com/56604066/74406989-d08d6b80-4de5-11ea-913f-4b0559f7bfb2.png">
<img width="1350" alt="Screen Shot 2020-02-12 at 10 18 21 PM" src="https://user-images.githubusercontent.com/56604066/74406977-cd927b00-4de5-11ea-9bca-0c196ab2f5e2.png">
Is this our expected behavior for when more than one column contains a very large value?